torch/testing/_internal/common_nn.py

```diff
@@ -640,7 +643,7 @@ def nllloss_no_reduce_test():
     return dict(
         fullname='NLLLoss_no_reduce',
         constructor=wrap_functional(
-            lambda i: F.nll_loss(i, t.type_as(i).long(), **kwargs)),
+            lambda i: F.nll_loss(i, t.type_as(i).long(), **kwargs)),  # type: ignore[arg-type]
```

Comment:
Why add the ignores for `**kwargs`, for readability of the test? I understand not replacing it with `reduction='none'` (which works fine), but you could replace with:
```
 F.nll_loss(..., reduction=kwargs['reduction'])
```
Then the test remains understandable (the `kwargs` above the `return` is what varies between tests), but the ignores aren't needed.

The advantage of that is that it adds to the `nll_loss` testing with type annotations (plus, the fewer ignores, the better).

Reply:
This applies to other ignores below as well.

Reply:
Done! I'll ping you if/when CI passes